android/BOINC/app/src/main/AndroidManifest.xml

```diff
@@ -38,7 +38,6 @@
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-    <uses-permission android:name="android.permission.READ_LOGS" />
```

Comment:
I'm not sure but does this mean that there will be no possibility to show logs for the user?

Reply:
This permission only pertains to the app reading the system logs.  The logs generated by the app should be fine.  On the debug build I'm running on my phone I have no issues seeing the boinc log messages.

Reply:
Here is the reference https://developer.android.com/reference/android/Manifest.permission.html#READ_LOGS

Nice and vague....

Reply:
Thank you! :thumbsup:

Reply:
Better answer: https://www.youtube.com/watch?feature=player_embedded&v=WDDgoxvQsrQ#t=1369s

When Jelly Bean was released, all apps get access to their logs without requiring additional permissions.  Additionally, apps are no longer allowed to ask for or use the READ_LOGS permission.

Since we only support API 19 and higher, this permission should be able to be safely removed.